### PR TITLE
fix: remove all wipp and polus-plugins code

### PR DESCRIPTION
**Pull request by camilovelezr**
This PR removes all wipp and polus-plugins code that was being used to convert from WIPP to ICT.
Now all of these features are part of [polus-tools-conversions](https://github.com/PolusAI/tools/tree/main/src/polus/tools/conversions)

### CODE REVIEW

File: src/ict/model.py [hunk 1]
```diff
@@ -86,6 +77,13 @@ def save_clt(self, cwl_path: StrPath, network_access: bool = False) -> Path:
             yaml.dump(self.to_clt(network_access), file)
         return Path(cwl_path)
 
+    def save_cwl(self, cwl_path: StrPath, network_access: bool = False) -> Path:
+        """Save the ICT as CommandLineTool to a file.
+
+        Alias for `save_clt`.
+        """
+        return self.save_clt(cwl_path, network_access)
```
Comment (nishaq503) [inline src/ict/model.py:90]: This looks like an infinite recursive call.

**Review by nishaq503 — CHANGES_REQUESTED** [review]
One of the methods looks like an infinite recursive call. If so, fix it. Otherwise, let me know and I will approve this PR.

File: src/ict/model.py (hunk 1 above)
Comment (camilovelezr) [reply src/ict/model.py:90]: It's an alias. See that one finishes in `clt` and the other in `cwl`.

**Comment by camilovelezr** [comment]
> One of the methods looks like an infinite recursive call. If so, fix it. Otherwise, let me know and I will approve this PR.

It is an alias and does not lead to an infinite recursive call

**Pull request merged**